.travis.yml

```diff
@@ -42,6 +42,9 @@ install:
   # Ensure postgres user has permissions
   - docker exec -it pgbuild /bin/bash -c "chown -R postgres:postgres /build/"
   - docker exec -it pgbuild /bin/bash -c "chown -R postgres:postgres /postgres/"
+before_script:
+  - sudo mkdir -p -m 0777 /cores
```

Comment:
Looks like cores are used here and in the core_pattern, but then /var/crash is used further below